src/config/utils.ts

```diff
@@ -5,3 +5,14 @@ import { ZERO_ADDRESS } from 'src/logic/wallets/ethAddresses'
 export const getNativeCurrencyAddress = (): string => {
   return ZERO_ADDRESS
 }
+
+// Template syntax returned from CGW is {{this}}
+export const replaceTemplateParams = (uri: string, data: string | Record<string, string>): string => {
+  const TEMPLATE_REGEX = /\{\{([^}]+)\}\}/g
+
+  if (typeof data === 'string') {
+    return uri.replace(TEMPLATE_REGEX, data)
+  }
```

Comment:
I wouldn't do that, it's much simpler to always pass an object with a named param, even if it's just one param.